pages/home/index.tsx

```diff
@@ -2,41 +2,44 @@
  * Copyright (c) HashiCorp, Inc.
  * SPDX-License-Identifier: MPL-2.0
  */
-
-import fs from 'fs'
-import path from 'path'
-import highlight from '@mapbox/rehype-prism'
-import rehypeSurfaceCodeNewlines from '@hashicorp/platform-code-highlighting/rehype-surface-code-newlines'
-import { serialize } from 'next-mdx-remote/serialize'
-import { MDXRemote, MDXRemoteSerializeResult } from 'next-mdx-remote'
 import { GetStaticPropsResult } from 'next'
-import remarkGfm from 'remark-gfm'
-import Button from 'components/button'
 import s from './style.module.css'
+import Nav from 'layouts/base/components/nav'
+import Footer from 'layouts/base/components/footer'
+import PeoplePage from 'pages/people'
 
-interface Props {
-	mdxSource: MDXRemoteSerializeResult
-}
+interface Props {}
+
+export default function IndexPage({}: Props): React.ReactElement {
+  return (
```

Comment:
Just an FYI, you didn't have to edit this file. Just making changes to `/people` would of been fine. Because `/people` and `/index` have slightly different styling I am just going to go by `/people`.

Reply:
Great, thanks :)